ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -1,5 +1,6 @@
 open Import
 module Version = Version
+module Diagnostics = Diagnostics
```

Comment:
I quite liked when you put for-test-only definitions in a separate module such as `Private` or `For_tests`. 

The same applies to `Diagnostics.equal_message` but it may get verbose `Ocaml_lsp_server.Private.Diagnostics.Private.equal_message` 🧐

Reply:
Yeah, I like that convention as well for public libraries. But in this case, ocaml-lsp-server is only a library to make it possible to add tests. So it seems a little unnecessary in this case.